jsonic3_commit.py

```diff
@@ -0,0 +1,80 @@
+"""
+Grabs data from the 'FAQ Content' CSV and turns it into nice JSON: a main faq object containing an array of Section objects, each Section Object in turn holding an array of Question objects consisting of question, answer and related link/s, as so:
+    [
+        {
+          'Q': 'What are the symptoms of COVID-19?',
+          'A': 'Symptoms of COVID-19 include coughing and shortness of breath. Additionally, a person showing two or more of the following symptoms may have the virus: fever, repeated shaking with chills, muscle pain, headache, sore throat, new loss of taste or smell.',
+          'link': 'https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html'
+        },
+    .
+    .
+    .
+    ]
+    etc
+In addition to an array of Question objects, each Section incorporates its title and a 'last updated' date value. For the Python intermediary stage (between CSV stream  and JSON) we will create a hierarchy of nested dictionaries and lists equivalent to the JSON objects and arrays.
+"""
+
+import requests
+import json
+import csv
+import datetime
+from typing import Iterable, Dict, List, Any
+
+
+# function to grab data from a google sheet + return as reader object
+def google_sheet_csv_data(sheet: str, gid: str) -> Iterable[List[str]]:
+    url = f'https://docs.google.com/spreadsheets/d/{sheet}/export'
+    response = requests.get(url, params={
+        'format': 'csv',
+        'id': sheet,
+        'gid': gid
+    })
+    response.raise_for_status()
+    return csv.reader(response.iter_lines(decode_unicode=True))
+
+
+# Pass faq Content sheet id and gid to function
+_reader = google_sheet_csv_data('1_wBXS62S5oBQrwetGc8_-dFvDjEmNqzqHwUeP-DzkYs',
+ '1318925039')
```

Comment:
This is an absurdly minor formatting nit, but when spreading arguments, list items, dictionary entries, etc. over multiple lines I try to stick to one of two conventions for better readability that are fairly common in Python:

1. Align all the items:

    ```suggestion
    _reader = google_sheet_csv_data('1_wBXS62S5oBQrwetGc8_-dFvDjEmNqzqHwUeP-DzkYs',
                                    '1318925039')
    ```

2. Give each item its own line and indent one level:

    ```suggestion
    _reader = google_sheet_csv_data(
        '1_wBXS62S5oBQrwetGc8_-dFvDjEmNqzqHwUeP-DzkYs',
        '1318925039'
    )
    ```